app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @url = "/tasks/#{@task[:id]}" %>
```

Comment:
Two potential ways to clean up this instance variable in the view: 
1. You could set the instance variable in the controller instead
2. You could pass a local variable into the partial